src/locales/i18n.js

```diff
@@ -0,0 +1,10 @@
+import * as defaultTranslations from "./en/translations.json";
+import * as I18nJs from "i18n-js";
+
+function t(key){
+  if (typeof I18n !== 'undefined'){ return I18n.t(key); }
+  I18nJs.translations["en"] = defaultTranslations["en"];
```

Comment:
Is this going to reassign `I18nJs.translations` every time we call a translation? I would assume this line would only happen once somewhere else and that this method would be 
```
function t(key){
  if (typeof I18n !== 'undefined'){ return I18n.t(key); }
  return I18nJs.t(key);
}
```

Reply:
I think i need to add a condition to load the default translation once time. Let me update this, thanks !